website/client/README.md

```diff
@@ -0,0 +1,20 @@
+#Running

```

Comment:
Not sure why this file got committed as a new one: I just renamed it from readme.md to README.md